test/lifecycle_test.dart

```diff
@@ -551,24 +551,6 @@ main() {
           expect(propTypeCheck, isA<JsBackedMap>());
         });
       });
-
-      group('props are received correctly without interop interfering with the values:', () {
```

Comment:
These were moved to the common factory tests so they run for all Dart components and Dart component HOCs, not just Dart function components